src/main/java/com/google/api/generator/gapic/composer/common/AbstractServiceSettingsClassComposer.java

```diff
@@ -110,7 +112,7 @@ public GapicClass generate(GapicContext ignored, Service service) {
             .setName(className)
             .setExtendsType(
                 TypeNode.withReference(
-                    FIXED_TYPESTORE
+                    getFixedTypeStore()
```

Comment:
Also  just basically a best practice thing for java - do not use protected fields, use protected setter/getter for them instead. This field can't be private, because it is accessed in the subclasses, so it must be either protected or have a protected getter, which is what is done here.

Reply:
Leaving open to revisit after the upcoming changes - if this becomes unneeded by the transport-specific composers, consider making it inaccessible to the subclasses.